Description: Fix issues with double VS qreal
 The code was attempting to pass double* to a function expecting qreal*
 resulting in compile errors on arm (where qreal is defined as float)
 this patch changes a couple of variable definitions so the types match
 up.
Bug-Debian: http://bugs.debian.org/??????

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- engauge-digitizer-5.0.orig/src/digitview.cpp
+++ engauge-digitizer-5.0/src/digitview.cpp
@@ -288,7 +288,7 @@ void DigitView::keyReleaseEvent(QKeyEven
 
 QPoint DigitView::convertZoom(QPoint p, bool to)
 {
-  double x, y;
+  qreal x, y;
   if (to)
     worldMatrix().map((double) p.x(), (double) p.y(), &x, &y);
   else
@@ -299,7 +299,7 @@ QPoint DigitView::convertZoom(QPoint p,
 
 QRect DigitView::convertZoom(QRect r, bool to)
 {
-  double xTL, yTL, xBR, yBR;
+  qreal xTL, yTL, xBR, yBR;
   if (to)
   {
     worldMatrix().map((double) r.left(), (double) r.top(), &xTL, &yTL);
